source/packages/modulo_core/tests/testCartesian.cpp

```diff
@@ -51,8 +51,8 @@ class ConsoleVisualizer : public modulo::core::Cell {
                                                                                                       desired_twist(std::make_shared<state_representation::CartesianTwist>("robot_test")) {}
 
   bool on_configure() {
-    this->add_subscription<geometry_msgs::msg::PoseStamped>("/robot_test/pose", this->robot_pose);
-    this->add_subscription<geometry_msgs::msg::TwistStamped>("/ds/desired_twist", this->desired_twist);
+    this->add_subscription<modulo::core::EncodedState>("/robot_test/pose", this->robot_pose);
```

Comment:
It might be worth adding signatures for that don't even require the template. For example, `add_state_subscription` and `add_state_publisher` would internally call `add_subscription<modulo::core::EncodedState>` and so on. This makes things even easier for the end user of a Cell-derived class, which is what we ultimately want.